src/app/pages/ArticlePage/types.ts

```diff
@@ -8,3 +8,13 @@ export type TimeStampProps = {
   firstPublished: number;
   lastPublished: number;
 };
+
+export type getHeadlineComponentProps = {
```

Comment:
Type naming convention
```suggestion
export type HeadlineComponentProps = {
```